ttnn/cpp/ttnn/operations/ccl/all_gather/all_gather.cpp

```diff
@@ -8,8 +8,8 @@
 
 namespace ttnn::operations::ccl {
 
-ttnn::Tensor ExecuteAllGather::invoke(const ttnn::Tensor& input_tensor, const uint32_t dim, const uint32_t num_links, const std::optional<ttnn::MemoryConfig>& memory_config, const std::optional<size_t> num_workers, const std::optional<size_t> num_buffers_per_channel) {
-    return ttnn::operations::ccl::all_gather(input_tensor, dim, num_links, memory_config, num_workers, num_buffers_per_channel);
+ttnn::Tensor ExecuteAllGather::invoke(const ttnn::Tensor& input_tensor, const uint32_t dim, const uint32_t num_links, const std::optional<ttnn::MemoryConfig>& memory_config, const std::optional<size_t> num_workers, const std::optional<size_t> num_buffers_per_channel, const all_gather_op::Topology topology) {
```

Comment:
This is still in the all gather namespace? It should just be ttnn::ccl::Topology

(same comment applies elsewhere in the commit still... Am I seeing old changes?)